<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="placeFiducialButton">
       <property name="text">
        <string>Add fiducial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="placeSpatialButton">
       <property name="text">
        <string>Add tracker position</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeFiducialButton">
       <property name="text">
        <string>Remove selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearFiducialsListButton">
       <property name="text">
        <string>Clear list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="fiducialsList">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Image Position</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fiducial ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tracker Position</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tracker ID</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
